packages/libs/eda/src/lib/core/components/visualizations/implementations/VolcanoPlotVisualization.tsx

```diff
@@ -343,7 +347,123 @@ function VolcanoPlotViz(props: VisualizationProps<Options>) {
   const plotNode = <VolcanoPlot {...volcanoPlotProps} ref={plotRef} />;
 
   // TODO
-  const controlsNode = <> </>;
+  const controlsNode = (
+    <div style={{ margin: '2em 1em' }}>
+      <SliderWidget
+        minimum={0}
+        maximum={1}
+        step={0.1}
+        value={vizConfig.markerBodyOpacity ?? 0.5}
```

Comment:
Got some feedback that the points are a little tough to see. Let's make the default 0.7 please!